seahorn/scripts/create_alloce_size_switch.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
why is it called `allocE`? 

rewrite in python

Reply:
don't generate into source file. Just generate new header file and include it where it is needed!